indico/modules/categories/serialize.py

```diff
@@ -18,6 +18,7 @@
 
 from io import BytesIO
 from itertools import ifilter
+from urlparse import urlparse
```

Comment:
i think usually we use `from werkzeug.urls import url_parse`  - it's pretty much the same but probably more unicode-friendly